app/src/main/java/com/example/drawingactivity/Hsv.kt

```diff
@@ -0,0 +1,55 @@
+package com.example.drawingactivity
+
+import android.graphics.Color
+
+/**
+ * A data class that contains color values.
+ * Contains: Hue, Saturation, and Value.
+ */
+data class Hsv(
+    var hue: Float,
+    var saturation: Float,
+    var value: Float
+) {
```

Comment:
Make a `toColor` function for this class

Reply:
Try to use it anywhere you make colors

Reply:
consolidated into a single function